lib/ansible/modules/cloud/amazon/cloudformation.py

```diff
@@ -343,7 +343,7 @@ def stack_operation(cfn, stack_name, operation):
             else:
                 ret.update({'changed': False, 'failed': True, 'output' : 'Stack not found.'})
                 return ret
-        elif stack['StackStatus'].endswith('_ROLLBACK_COMPLETE'):
+        elif stack['StackStatus'].endswith('ROLLBACK_COMPLETE'):
```

Comment:
Hmm. Should `UPDATE_ROLLBACK_COMPLETE` and `ROLLBACK_COMPLETE` return in the same manner? I can't think of a reason not, but want to make sure we acknowledge it and linking to [the list of possible states](http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-describing-stacks.html#w1ab2c15c17c21c13). Perhaps a comment indicating these states are covered? Also, can you fix the spacing on `'failed' :True`? Not your fault, but I see it now.